src/NetAnalyzers/CSharp/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CSharpGenericInterfaceShouldAlsoBeImplemented.Fixer.cs

```diff
@@ -1,17 +1,17 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System.Composition;
-using Microsoft.CodeQuality.Analyzers.ApiDesignGuidelines;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeQuality.Analyzers.ApiDesignGuidelines;
 
 namespace Microsoft.CodeQuality.CSharp.Analyzers.ApiDesignGuidelines
 {
     /// <summary>
-    /// CA1010: Collections should implement generic interface
+    /// CA1010: Generic interfaces should be implemented
     /// </summary>
     [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
-    public sealed class CSharpCollectionsShouldImplementGenericInterfaceFixer : CollectionsShouldImplementGenericInterfaceFixer
+    public sealed class CSharpGenericInterfacesShouldAlsoBeImplementedFixer : GenericInterfacesShouldAlsoBeImplementedFixer
```

Comment:
The existing type names were chosen to exactly match the type names in legacy implementation so it acts as an easy reference point. Additionally, the docs also use the prior name: https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1010?view=vs-2019.